pyroSAR/snap/auxil.py

```diff
@@ -191,6 +192,13 @@ def gpt(xmlfile):
     input is a readily formatted workflow xml file as created by function geocode in module snap.util
     """
 
+    # Test if SNAP is installed:
```

Comment:
function gpt needs the gpt command line tool, thus ExamineExe should also check for it and not the four different snap executable options